src/material/core/theming/_palette.scss

```diff
@@ -704,6 +705,7 @@ $mat-dark-theme-background: (
   disabled-button-toggle: black,
   unselected-chip: map_get($mat-grey, 700),
   disabled-list-option: black,
+  tooltip: rgba(map_get($mat-grey, 700), 0.9),
```

Comment:
This will throw a compilation error if `$mat-grey` 700 resolves to a CSS variable.

Reply:
I moved opacity inside _tooltip-theme.scss
so mat-color will handle css variables